packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/datacentre-general-information-tile/DatacentreGeneralInformationTile.spec.tsx

```diff
@@ -78,8 +79,8 @@ describe.skip('DatacentreGeneralInformationTile component unit test suite', () =
     const { getByText } = render(
       <QueryClientProvider client={queryClient}>
         <DatacentreGeneralInformationTile
-          vcdOrganization={vcdOrg}
-          vcdDatacentre={datacentre}
+          vcdOrganization={vcdOrg as VCDOrganization}
+          vcdDatacentre={datacentre as VCDDatacentre}
```

Comment:
Maybe declare the constante on declaration with this type ?

Reply:
They are mocked so they don't correspond exactly to the types probably
No need to include a full typed mock here but we don't want the warning either